hawk/app/assets/stylesheets/authentication/_content.scss

```diff
@@ -1,13 +1,18 @@
 // Copyright (c) 2009-2015 Tim Serong <tserong@suse.com>
 // See COPYING for license.
 
+.primitive {
+  line-height: 1.8em;
+  font-size: 1.3em;
+}
+
 #content {
   width: 300px;
 
   &.darkbar {
-    background-color: #2b2b2b;
+    background-color: #05555E;
     border-radius: 0px 0px 8px 8px;
-    border: solid 1px #333;
+    border: solid 0px #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`